FirebaseRemoteConfig/Tests/Sample/Podfile

```diff
@@ -0,0 +1,18 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'RemoteConfigSampleApp' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+  pod 'FirebaseAnalytics'
+  pod 'FirebaseCore', :path => '../../../'
```

Comment:
Would it better to qualify the podspec instead of an open path here?

Here and below!